.github/workflows/windows.yml

```diff
@@ -256,6 +256,9 @@ jobs:
       INSTALL_TEST_DIR: "${{ github.workspace }}\\install\\tests"
       LAYER_TESTS_INSTALL_DIR: "${{ github.workspace }}\\install\\tests\\layer_tests"
       PYTHON_STATIC_ARGS: -m "not dynamic_library and not template_plugin"
+    strategy:
+      matrix:
+        test_group: [mo, api, pytorh, tf, tf_lite, ovc, onnx]
```

Comment:
it's too expensive to run each group on dedicated runner.

@akashchi will try to parallel via built-in pytest `-n` parameter.
https://www.tutorialspoint.com/pytest/pytest_run_tests_in_parallel.htm

Reply:
I've already tried it too, to properly run it we would need to modify tests.

Reply:
> to properly run it we would need to modify tests.

do you have more details?
I suppose it's more proper way